services-core/src/price_finding/optimization_price_finder.rs

```diff
@@ -292,7 +298,8 @@ impl PriceFinding for OptimisationPriceFinder {
         let solver_type = self.solver_type;
         // The solver expects the fee amount as the total paid fees. Half of the paid fees are
         // burned and half earned.
-        let min_avg_paid_fee_per_order = 2 * self.economic_viability.min_average_fee().await?;
+        let min_avg_fee = 2 * self.economic_viability.min_average_fee().await?;
```

Comment:
Unrelated to this but maybe we should expose two methods `min_average_trading_fee`, `min_average_solver_reward` (one could have a default implementation calling the other) to make the difference between the two more clear.